categories/categories.css

```diff
@@ -0,0 +1,345 @@
+@import url('https://fonts.googleapis.com/css2?family=Redressed&family=Roboto:wght@300&family=Satisfy&family=Shizuru&display=swap');
+
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100&family=Mukta:wght@300&family=Palette+Mosaic&family=Prompt:wght@300&family=Redressed&family=Roboto:wght@300&family=Satisfy&family=Shizuru&display=swap');
+
+body{
+  
+    font-family: 'Inter', sans-serif;
+    font-family: 'Mukta', sans-serif;
+    background-image: url("blue-copy-space-digital-background_23-2148821698.webp");
+    background-repeat: no-repeat;
+    background-size: cover;
+   
+box-sizing: border-box;
+margin: 0px;
+padding: 0px;
+/* background-color: #e056fd;
+background-image: linear-gradient(315deg, #e056fd 0%, #000000 74%); */
+}
+
+
+
+* {
+
+--primary-color: #ecfeff ;
+--secondary-color : #67e8f9;
+--hover-color: #e056fd ;
+}
+
+.container{
+    color: var(--primary-color);
+    width: 100vw;
+    text-decoration: none;
+    display: grid;
+    font-weight: 400;
+    grid-template-rows: 200px 1fr 200px; 
+ 
+    grid-template-areas: 
+    "nav nav nav"
+    "main main main "
+    " footer footer footer"
+    " footer footer footer";
+}
+
+nav{
+    display: flex;
+    gap: 20px;
+    width: 100vw;
+    height: 16vw;
+    grid-area: nav;
+    
+
+}
+
+nav h2{
+    position: relative;
+  
+    font-size: 4rem;
+    line-height: 1.2rem;
+    letter-spacing: 0.125rem;
+}
+
+.imp-text{
+    font-family: 'Inter', sans-serif;
+font-family: 'Mukta', sans-serif;
+font-family: 'Palette Mosaic', cursive;
+font-family: 'Redressed', cursive;
+font-family: 'Roboto', sans-serif;
+font-family: 'Satisfy', cursive;
+font-family: 'Shizuru', cursive;
+}
+
+.current-pg{
+    color: var(--secondary-color);
+}
+
+
+
+nav ul{
+    margin-left: auto; 

```

Comment:
Keep the indentation same everywhere. Ideally, 2 spaces in HTML/CSS according to Google Style Guide